modules/ROOT/pages/latest-overview-concept.adoc

```diff
@@ -37,19 +37,28 @@ _Applications_ are external services that consume APIs. For details about applic
+* *DataGraph Administration*
++
+Anypoint DataGraph enables you to unify all the data within your application network in a unified schema. For details, see xref:datagraph::index.adoc[Anypoint DataGraph Overview].
 * *Analytics*
 +
 API analytics provide insight into how your APIs are being used and how they are performing. For details, see xref:viewing-api-analytics.adoc[Reviewing Analytics Usage].
 
 <3> The *Manage API* button. Enables you to add an existing API instance from Exchange, to upload a RAML/OAS/SOAP definition, or to import a configuration ZIP file exported from API Manager. For details, see xref:getting-started-proxy.adoc[Getting Started with Managing an API].
 
-<4> The *Promote from environment* button. Enables you to promote an API from any environment to the current environment. For details, see xref:promote-api-task.adoc[Promoting an API Instance to Another Environment].
+<4> The *Environment information* button, available only to administrator users. Enables administrators to display a dialog with information about the current environment, such as environment credentials. Use environment credentials to provision the Anypoint Service Mesh adapter or to configure Studio to sync with your environment.
+
+<5> The *Promote from environment* button. Enables you to promote an API from any environment to the current environment. For details, see xref:promote-api-task.adoc[Promoting an API Instance to Another Environment].
 
-<5> Search. Enables you to search for managed APIs via the API search field. Note that searches are case-sensitive. Filter search results by selecting *All*, *Favorites*, or *Active*.
+<6> Search. Enables you to search for managed APIs via the API search field. Note that searches are case-sensitive. Filter search results by selecting *All*, *Favorites*, or *Active*.
 
-<6> The *Environment information* button, available only to administrator users. Enables administrators to display a dialog with information about the current environment, such as environment credentials. Use environment credentials to provision the Anypoint Service Mesh adapter or to configure Studio to sync with your environment.
+<7> The tracking registration status of each API: *Active*, *Inactive*, *Deploying*, or *Unregistered*.  
++
+A status of *Unregistered* means that Anypoint Platform has never tracked the endpoint for this API version. Either you have entered a URL for an API or proxy that is not yet deployed, or you have declared an endpoint that is hosted somewhere other than an API gateway and need to proxy that endpoint so that the platform can track it.
```

Comment:
> Either you have entered a URL for an API or proxy that is not yet deployed, or you have declared an endpoint that is hosted somewhere other than an API gateway and need to proxy that endpoint so that the platform can track it.

This is a bit confusing.

Unregistered means that the API or Proxy have never talked at least once with the platform.